delta/commandsEvents/partitions/AddPartition/PartitionAdded.adoc

```diff
@@ -9,7 +9,9 @@ This client is now subscribed to any changes to <<PartitionAdded.newPartition>>
 
 [horizontal]
 .Parameters
-[[PartitionAdded.newPartition, `newPartition`]]newPartition:: <<chunkType>>
+[[PartitionAdded.newPartition, `newPartition`]]newPartition:: <<shallowChunkType>>
```

Comment:
I see no reason to introduce a new type for this, we can (as we already did) use the << chunkType >> for this, with the explanation we already had, saying that the containments, references and annotations are empty.
Feels like overkill.

Reply:
At all other places I found it used in the spec, I don't think it should be there, as I said  in several comments.